Cargo.toml

```diff
@@ -116,3 +116,6 @@ crate-type = ["bin"]
 [[example]]
 name = "callback"
 crate-type = ["bin"]
+
+[patch.crates-io]
+wabt-sys = { git = "https://github.com/pepyakin/wabt-rs.git" }
```

Comment:
Just created this issue in WABT to publish a new version in crates.io:
https://github.com/pepyakin/wabt-rs/issues/65

Once is merged we should be ready to go!

Reply:
wabt-sys 0.7.1 is out.

Reply:
@igrep as soon as this line is updated we will merge! 🎉

Thanks @pepyakin for the quick release!